include/cinder/Log.h

```diff
@@ -116,36 +125,66 @@ class LoggerFile : public Logger {
 	void		ensureDirectoryExists();
 
 	fs::path		mFilePath;
+	bool			mAppend;
+	std::ofstream	mStream;
+};
+
+//! LoggerFileRotating will write to a specified file, either appending to or overwriting that
+//! file.  The filename will be re-evaluated during the first logging event occuring past midnight.
+class LoggerFileRotating : public LoggerFile {
+  public:
+	//! LoggerFileRotating constructor that creates a rotating log file.  The filename will be
+	//! re-evaluated when the first logging event occurs after midnight.
+	//! File appending is configurable via \p appendToExisting.  \p formatStr will be passed to
+	//! [strtime](http://www.cplusplus.com/reference/ctime/strftime/) to determine the file name.
```

Comment:
About the length function, I think we could refer to the ctime function as just `strftime()` (and make sure to get the spelling right), users can search for docs on it via whatever manner they like. I also agree though, detailed docs on parameters in the header like this is okay.